pages/linux/wodim.md

```diff
@@ -1,14 +1,13 @@
 # wodim
 
-> Command for record/burning data using a optical drive to CD or DVD.
-> This command is also called cdrecord on some systems, but it is wodim underneath.
-> Some usage of wodim can cause destructive actions, such as erasing all the data on a disc.
+> Command (aliased as `cdrecord` on some systems) for recording data to CDs or DVDs.
```

Comment:
`aliased as` -> `aliased to`?

Reply:
That suggests the wrong direction of the relationship, but it may be due to English nuances. For example, I feel it's appropriate to say "`foo` is a link to `bar`" or "`bar` is linked from `foo`" -- always making it clear that "foo" is the secondary object, and "bar" the primary one.

"Aliased to" kind of sounds like "linked to" to my ears, but again, I'm not a native English speaker, so I'll defer to whatever the community decides on this one.

Reply:
It's a really small point, so either will be just fine :-)

Reply:
I guess my arguments didn't sway you, then :P let's agree to disagree?

Reply:
Yep :P